### DISCUSSION TITLE
Arena Model Improvements to behave more like LYSMS

### DISCUSSION THREAD

**Discussion by notasquid1938** [open-webui/open-webui#6647] [Ideas]
The current Arena Model implementation as described in #6538 doesn't make a ton of sense:

"It will randomly select a model from the collection to provide an answer to your query. Once you rank the response (use like or dislike button), the Arena model will reveal which specific model generated it, making this process a blind test. You can regenerate the response to obtain another one from a different model for comparison (note that the second response could come from the same model as the first)."

The leaderboard uses ELO to rank models, but they're not actually being compared to each other. Each response is evaluated in a vacuum and the user has to refer back across regenerated responses to dictate a winner. For instance, I generated a response twice to a question and liked both so I gave them both a thumbs up. However, now my leaderboard shows the model that was liked earlier as having "lost" which doesn't make a ton of sense. 

It would make more sense from a user and UI perspective if when a prompt was given two models were picked at random to generate a response side by side, and the user clicked on which response they liked better or if it was a tie.

**Comment by tjbck** [comment]
FYI, you can already achieve what you described by adding two models in the same chat.

![image](https://github.com/user-attachments/assets/1ca10f86-eecf-465d-b7cb-ac0bc0efa98c)

**Reply by martinng01** [reply]
I think a better implementation would be the ability to combine two models into a single grouped model that users can select for the Chatbot Arena style setup.

Currently the user has to add the second model by themselves which is not ideal.

**Reply by GrayXu** [reply]
After scoring, can we directly tell the user the specific name of this anonymous model?

**Comment by GrayXu** [comment]
Multiple evaluations in a single reply will still cause the problem of multiple duplicate scores.
